email-templates/201012.html

```diff
@@ -0,0 +1,105 @@
+<html>
+  <body>
+    <p>
+      <b class="TODO">Någon fräsig ingress</b>
+    </p>
+    <p>
+      <ul>
+        <li> Just nu pågår Hacktoberfest, ett månadslångt evenemang som
+             uppmärksammar open source. Om du gör fyra pull requests till
+             (medverkande) projekt på Github får du en Hacktoberfest-tröja!
+             (Eller så kan du välja att plantera ett träd.) Läs mer på
+             <a href="http://hacktoberfest.com">http://hacktoberfest.com</a> </li>
+
+        <li> Den 7:e november är det återigen dags för NCPC! NCPC är en
+             heldagstävling i programmering där lag om upp till tre personer
+             tillsammans löser programmeringsproblem av varierande
+             svårighetsgrad. Universitets bästa lag går vidare till NWERC
+             (Northwestern Europe Regional Contest). Vanligtvis är NCPC en
+             tävling som helt går på plats, men i år kommer det gå att tävla
+             både på plats och hemifrån. Se <a
+             href="https://lithekod.se/ncpc/se">https://lithekod.se/ncpc</a>
+             för mer information samt hur du anmäler dig till själva tävlingen. </li>
+
+        <li> Vill du bygga en Arudino, löda och dricka öl (eller annan valfri
+             dryck med eller utan alkohol)? Om så är fallet, kom på Lödöl med
+             MindRoad den 12:e november! Du kommer att få chansen att löda en
+             Arduino, en display och ett par sensorer på ett mönsterkort som du
+             sedan kan programmera själv. Anmäl dig på <a href="https://lithekod.se/lodol/">lithekod.se/lodol</a>
+             senast <b class="TODO">DATUM</b>. </li>
+
+        <li> Vårt traditionsenliga Fall Game Jam kommer gå 13-15 november. Som i
+             våras kommer det hållas på distans i <b class="TODO">LiU Game Jams
+             Discordserver (LÄNK)</b> Se till att <b class="TODO">göra något för
+             att bli uppdaterad</b>. </li>
+
+        <li> Höstmötet kommer att hållas i
+             vår <a href="https://discord.gg/2BkvvVx">discordserver</a> tisdag
+             24:e november kl 17.15. Samtliga medlemmar får närvara med
+             rösträtt.
+             Se <a href="https://lithekod.se/posts">https://lithekod.se/posts</a>
+             för mer information. </li>
+      </ul>
+    </p>
+    <p>
+    --<br>
+    LiTHe kods styrelse
+    </p>
+    <p>
+      Om du inte vill få fler informationsutsick från oss, <a
+      href="{unsubscribe_link}">avsluta din prenumeration här</a>.
+    </p>
+
+    <hr>
+
+    <p>
+      <b class="TODO">Någon fräsig ingress</b>
+    </p>
+    <p>
+      <ul>
+        <li> Hacktoberfest is running right now, a month-long event celebrating
```

Comment:
```suggestion
        <li> Do not miss Hacktoberfest, a month-long event celebrating
```
Jag vet inte riktigt hur det borde ändras, men det borde ändras på något sätt.